server/views/layout.njk

```diff
@@ -91,6 +91,25 @@
   <span class="govuk-visually-hidden" id="{{ pageId }}"></span>
 {% endblock %}
 
+{% block footer %}
+  {{ govukFooter({
+    meta: {
+      items: [
+        {
+          href: "/about",
+          text: "About",
+          attributes: { 'target': '_blank' }
+        },
+        {
+          href: "/contact",
+          text: "Contact",
+          attributes: { 'target': '_blank' }
+        }
+      ]
```

Comment:
Accessibility page still to come?

Reply:
And still no actual content for the About us, so a placeholder for now.